compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmTerminatorGen.java

```diff
@@ -604,8 +611,11 @@ private void genJICallTerm(JIMethodCall callIns, int localVarOffset) {
             this.mv.visitVarInsn(ALOAD, localVarOffset); // load the strand
             // load the current Module
             mv.visitFieldInsn(GETSTATIC, this.moduleInitClass, CURRENT_MODULE_VAR_NAME, GET_MODULE);
+            // load function name
+            mv.visitLdcInsn(func.name.getValue());
+            loadFuncPathParams(mv, (BInvokableTypeSymbol) func.type.tsymbol);
             mv.visitMethodInsn(INVOKESPECIAL, BAL_ENV, JVM_INIT_METHOD,
-                               INIT_BAL_ENV, false);
```

Comment:
Can we remove this constant?

Reply:
This is used in `generatePlatformIns` in `JVMInstructionGen`